src/tokens/Avatar/Avatar.cairo

```diff
@@ -13,11 +13,13 @@ from openzeppelin.security.initializable.library import Initializable
 from openzeppelin.access.ownable.library import Ownable
 from openzeppelin.upgrades.library import Proxy
 
-from src.tokens.Avatar.library import create_tokenURI, create_data
+from src.tokens.Avatar.library import create_tokenURI
 from src.tokens.Avatar.progress import get_progress
 from src.util.str import string, str_concat, str_from_literal
 
-const EXPERIENCE_CONTRACT = 0x00c62540e9a10c47a4b7d8abaff468192c66f2d1e979f6bade6e44bf73995982;
```

Comment:
we need to remove this to support deploying on mainnet, since the experience contract deployment address will be different